code/procs/jobprocs.dm

```diff
@@ -479,8 +539,9 @@
 			src.equip_new_if_possible(JOB.slot_belt, slot_in_backpack)
 		else
 			src.equip_new_if_possible(JOB.slot_belt, slot_belt)
-			if (src.traitHolder && src.traitHolder.hasTrait("immigrant") && istype(src.belt, /obj/item/device/pda2))
-				del(src.belt) //UGHUGHUGHUGUUUUUUUU
+			if (src.traitHolder && istype(src.belt, /obj/item/device/pda2))
+				if (src.traitHolder.hasTrait("immigrant") || src.traitHolder.hasTrait("pilot"))
+					del(src.belt)
```

Comment:
Is it really intended that you start without a PDA? That seems a bit weird.

Reply:
Yep. Prevents Port-a-Sci/Med usage and nudges you to use your headset, if you have one.